.gitignore

```diff
@@ -1,5 +1,6 @@
 *.rbc
 capybara-*.html
+.idea
```

Comment:
Can you add a comment to reflect that `.idea` files are auto-generated by RubyMine? I wasn't sure what this was for.